### ISSUE TITLE
Why the reconstructed image is very blurry

### ISSUE THREAD

**Issue by zerolfq** [google-research/multinerf#78]
Why the reconstructed image is very blurry
![color_000](https://user-images.githubusercontent.com/49546423/205050999-fd3650e2-53e8-424f-9374-7f3c38594c2b.png)

**Comment by zerolfq** [comment]
I execute according to the readme command, and the data set uses the data downloaded in mip Nerf360

**Comment by jonbarron** [comment]
How long did you train for?

**Comment by zerolfq** [comment]
> How long did you train for?

250000 iterations，Use the configuration of 360.gin，Config.batch_size=16

**Comment by jonbarron** [comment]
The batch size needs to be way bigger.

**Comment by Riga27527** [comment]
Hi, @jonbarron . I'm now using blender_refnerf.gin, but due to memory limitations, the batch_size is set to 2048, how many rounds of training would be better? Is 500,000 rounds enough?

**Comment by jonbarron** [comment]
I'd follow the "linear scaling rule" when changing batch size. You should be scaling up the number of iterations and scaling down the learning rate so that the model sees the same number of pixels as it had previously.
